config/urls.py

```diff
@@ -65,9 +66,11 @@
 
 urlpatterns = [
     url(r'^api/auth/token/$', obtain_auth_token),
-    url(r'^api/auth/status/$', AuthStatusView.as_view()),
     url(r'^api/auth/logout/$', LogoutView.as_view()),
     url(r'^api/auth/user/$', AuthUserView.as_view()),
+    url(r'^api/auth/user/verify_mail/$', VerifyMailView.as_view()),
+    url(r'^api/auth/user/resend_verification/$', ResendVerificationView.as_view()),
```

Comment:
I would agree with your idea to make it less nested (which also makes it more consistent with `reset_password`.

Reply:
I'll move them one level up.